docs/source/rule-testing.rst

```diff
@@ -76,6 +76,84 @@ For example, to replace a time based field with ``last_hour``:
   }
 
 
+Validating Log Schemas
+~~~~~~~~~~~~~~~~~~~~~~
+
+In some cases, there may be logs coming into StreamAlert that are of a known type, but you do not have a current need to apply rule
+logic to them. However, it is still advantageous to add schemas for these logs and *test* to ensure your new schemas are valid.
+
+This is possible through the addition of log schema(s) to ``conf/logs.json`` and creation of test record(s) in ``tests/integration/rules/``
+using known samples of logs (without actually adding a corresponding rule). Running the ``manage.py`` script with the ``validate-schemas``
+option will iterate over all json test files and attempt to match records within them to a defined log schema.
+
+To simply run schema validation on all test files:
+
+.. code-block:: bash
+
+  $ python manage.py validate-schemas
+
+
+To run schema validation on a specific test file within ``tests/integration/rules/``:
+
+.. code-block:: bash
+
+  $ python manage.py validate-schemas --test-files <test_rule_file.json>
+
+Or:
+
+.. code-block:: bash
+
+  $ python manage.py validate-schemas --test-files <test_rule_file>
+
+
+Example of running schema validation on two valid test files:
```

Comment:
unclear what you're looking for here? just `Schema validation on two valid test files:`?